olp-cpp-sdk-dataservice-read/src/VersionedLayerClientImpl.cpp

```diff
@@ -292,16 +303,18 @@ client::CancellationToken VersionedLayerClientImpl::PrefetchPartitions(
               request.GetPriority());
         });
 
+        std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```

Comment:
Quite a big delay. Why we need it?

Reply:
Please remove it, it's not needed.

Reply:
ouch, that was tmp code to help reproduce the issue.
Removed :-)